### PR TITLE
Create Movie: model, DAO, impl.

### PR DESCRIPTION
**Pull request by fyarkiy**
Add Injector (old).
create hibernate.cfg.xml, adjust pom to have dependency for Hibernate

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,6 +7,75 @@
...
-
-</project>
\ No newline at end of file
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (MaliukDaria) [inline pom.xml]: Checklist: Don’t add to pom.xml dependencies that you don’t use

File: src/main/java/com/cinema/dao/impl/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            transaction.commit();
+            movie.setId(utemId);
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Not able to add movie to data base", e);
+        } finally {
+            session.close();
```
Comment (MaliukDaria) [inline src/main/java/com/cinema/dao/impl/MovieDaoImpl.java]: What if something went wrong inside methods getSessionFactory() or openSession();?

File: src/main/java/com/cinema/dao/impl/MovieDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        }
+    }
+
+    @Override
+    public List<Movie> getAll() {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllQueryMovie = session.createQuery("from Movie ", Movie.class);
+            return getAllQueryMovie.getResultList();
+
```
Comment (MaliukDaria) [inline src/main/java/com/cinema/dao/impl/MovieDaoImpl.java]: empty line

File: src/main/java/com/cinema/model/Movie.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
+package com.cinema.model;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
```
Comment (MaliukDaria) [inline src/main/java/com/cinema/model/Movie.java:10]:
```suggestion
@Entity
@Table(name = "movie")
```

File: src/main/java/com/cinema/model/Movie.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String descritpion;
+
+    public Movie() {
+    }
```
Comment (MaliukDaria) [inline src/main/java/com/cinema/model/Movie.java]: Checklist: "Don’t create default constructor when it is not needed"

File: src/main/resources/hibernate.cfg.xml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        <property name="connection.url">jdbc:mysql://localhost/cinema?serverTimezone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">1234</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="com.cinema.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (MaliukDaria) [inline src/main/resources/hibernate.cfg.xml:18]: add empty line

Comment (fyarkiy) [reply src/main/resources/hibernate.cfg.xml:18]: it's there - otherwise it will be stop-sign:)

Comment (MaliukDaria) [reply src/main/resources/hibernate.cfg.xml:18]: sorry)

File: src/main/java/com/cinema/dao/impl/MovieDaoImpl.java (hunk 2 above)
Comment (dmytro-hryhoruk) [reply src/main/java/com/cinema/dao/impl/MovieDaoImpl.java]: agree

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**